tests/test_builder_containerbuild.py

```diff
@@ -715,6 +716,7 @@ def test_createSourceContainer_failure_source(self, reason, expected_exc_type):
         if reason == 'signal_cancelled':
             task._incremental_upload_logs = \
                 lambda pid: os.kill(os.getpid(), signal.SIGINT)
+            flexmock(task).should_receive('_upload_logs_once').once().and_return(True)
```

Comment:
I'm puzzled here `.and_return(True)` that function doesn't return anything (i.e. `None` in py world)

Reply:
yeah changed to None, either way it is just to mock that methods to check it was called